src/main/java/io/mosip/print/service/impl/UinCardGeneratorImpl.java

```diff
@@ -125,22 +124,18 @@
 			pdfSignatured = Base64.decodeBase64(signatureResponseDto.getData());
 
 		} catch (IOException | PDFGeneratorException e) {
-			printLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
-					"", PlatformErrorMessages.PRT_PRT_PDF_NOT_GENERATED.name() + e.getMessage()
+			printLogger.error( e.getMessage()
 							+ ExceptionUtils.getStackTrace(e));
 			throw new PDFGeneratorException(PDFGeneratorExceptionCodeConstant.PDF_EXCEPTION.getErrorCode(),
 					e.getMessage() + ExceptionUtils.getStackTrace(e));
 		} 
 			  catch (ApisResourceAccessException e) {
 					e.printStackTrace();
-					printLogger.error(LoggerFileConstant.SESSIONID.toString(),
-			 LoggerFileConstant.REGISTRATIONID.toString(), "",
-						PlatformErrorMessages.PRT_PRT_PDF_SIGNATURE_EXCEPTION.name() + e.getMessage()
+					printLogger.error(PlatformErrorMessages.PRT_PRT_PDF_SIGNATURE_EXCEPTION.name() + e.getMessage()
 			 + ExceptionUtils.getStackTrace(e)); throw new PDFSignatureException(
```

Comment:
this is not the correct way to throw.  use as below. Also, change in all places where you are throwing an exception
`throw new PDFSignatureException(e);`

Reply:
taken care